taxonomies/food/ingredients.txt

```diff
@@ -29270,6 +29275,10 @@ sv: majsflingor
 # 22 products in 5 languages @2018-09-23
 # ingredient/fr:pétales-de-maïs 180 products in 8 languages @2018-09-24
 
+< en:corn
+en: corn crisps
+sv: majskrisp
```

Comment:
same as previous comment. Need to add both crips and krisp under:
```
en: chips, slices
cs: lupínků
de: chipsstücke
es: copos de
fi: lastu
fr: chips de
it: fettine di
ja: チップス, チップ
nl: chips
pl: chipsy, płaty, płaty z, płaty z, chipsy z, chipsów
```

Reply:
In Swedish, “‐krisp” usually refer to puffed products, so added “krisp” as another `sv` synonym there.